masterlist.yaml

```
@@ -4315,6 +4315,28 @@ plugins:
         util: '[FO3Edit v4.0.3g](https://www.nexusmods.com/fallout3/mods/637)'
         nav: 16
 
+  - name: 'CubeExperimental (EN).esm'
+    url:
+      - link: 'https://www.nexusmods.com/fallout3/mods/6270/'
+        name: 'Cube Experimental'
+    msg:
+      - <<: *obsolete
+        subs: [ '[Cube Experimental Chakys Fix Version 1_0_4](https://www.nexusmods.com/fallout3/mods/6270/)' ]
+        condition: '(not checksum("CubeExperimental (EN).esm", 1E0B90CB) and not checksum("CubeExperimental (EN).esm", 5E1CD9C8))'
```

I would only check for the outdated versions uncleaned CRC, as it's more reliable.
Each update to xEdit could potentially produce a different CRC after cleaning.

```suggestion
        condition: 'checksum("CubeExperimental (EN).esm", CFEB2A0D)'
```